sample/screenshots.gradle

```diff
@@ -17,7 +17,9 @@ ext {
 }
 
 shot {
+    tolerance = 0.05
     runInstrumentation = Boolean.valueOf(getArgument("runInstrumentation", "true"))
+    showOnlyFailingTestsInReports = true
```

Comment:
Will we still be able to get hold of screenshots that pass? I assume so, just wanna double check

Reply:
We'll have the screenshots we need in the baseline folder